server/controllers/api/user/password/index.js

```diff
@@ -0,0 +1,69 @@
+const { handleErrorResponse } = require('../../../utils/errorHandlers.js');
+const logger = require('winston');
+const db = require('../../../../models');
+const { auth: { masterPassword } } = require('../../../../../config/siteConfig.js');
```

Comment:
This seems like it could be a security vulnerability?

Reply:
As discussed in standup, the siteConfig will not be kept in version control (it is pulled from www.spee.ch's config which is gitignored).  I'm going to merge.